src/cascadia/TerminalSettingsEditor/ColorSchemeViewModel.cpp

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "ColorSchemeViewModel.h"
+#include "ColorSchemeViewModel.g.cpp"
+
+#include <LibraryResources.h>
+#include "..\WinRTUtils\inc\Utils.h"
+
+namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
+{
+    static constexpr uint8_t ColorTableDivider{ 8 };
+    static constexpr uint8_t ColorTableSize{ 16 };
+
+    static constexpr std::wstring_view ForegroundColorTag{ L"Foreground" };
+    static constexpr std::wstring_view BackgroundColorTag{ L"Background" };
+    static constexpr std::wstring_view CursorColorTag{ L"CursorColor" };
+    static constexpr std::wstring_view SelectionBackgroundColorTag{ L"SelectionBackground" };

```

Comment:
These could probably be taken out and put in a common header since `ColorSchemes.cpp` also use them